src/libraries/System.Private.CoreLib/src/System/Action.cs

```diff
@@ -24,7 +26,7 @@ namespace System
     public delegate TResult Func<in T1, in T2, in T3, in T4, in T5, in T6, in T7, out TResult>(T1 arg1, T2 arg2, T3 arg3, T4 arg4, T5 arg5, T6 arg6, T7 arg7);
     public delegate TResult Func<in T1, in T2, in T3, in T4, in T5, in T6, in T7, in T8, out TResult>(T1 arg1, T2 arg2, T3 arg3, T4 arg4, T5 arg5, T6 arg6, T7 arg7, T8 arg8);
 
-    public delegate int Comparison<in T>(T x, T y);
+    public delegate int Comparison<in T>([AllowNull] T x, [AllowNull] T y);
```

Comment:
Good catch.  We need to update the ref assembly accordingly:
https://github.com/dotnet/runtime/blob/ca6adaac796a19d70e4361a9b2a8d090c2f27637/src/libraries/System.Runtime/ref/System.Runtime.cs#L611